README.md

```diff
@@ -25,7 +25,12 @@ It is super easy to use App Center. Have a look at our [get started documentatio
 
 We are looking forward to your contributions via pull requests.
 
-To contribute to the SDK, you need a machine that runs macOS and the latest stable Xcode release. In addition, the SDK workspace requires [Jazzy](https://github.com/realm/jazzy) to generate documentation. 
+To contribute to the SDK, please
+
+* install Xcode 10.1 and Xcode 8.3.3 in parallel (information on how to do that can be found [in this blogpost](https://medium.com/@hacknicity/working-with-multiple-versions-of-xcode-e331c01aa6bc)).
```

Comment:
Since I ran into this problem, we should mention that 8.x can't be used on Mojave.  Or is it just the older MacBook Pros that it can't be used on?  What if someone can't install multiple versions for this reason, should we just say that they can't build the SDK for the correct architectures?  I guess I don't really know what path to development we want to give people in those situations

Reply:
It's Mojave.

Reply:
Right, so what about people on Mojave?  Are we just saying that they can't contribute until arm64e is out of preview?

Reply:
In my opinion, if someone is contributing, they don't need to build the SDK the same way we do at release time with CI. So I actually think we should tell people to use Xcode 10.1.

Reply:
That seems to be a good compromise.  We can also have instructions for testing out the arm64e preview for people who want to do it.